data.json

```diff
@@ -467,4 +473,5 @@
     "imageUrl": "https://user-images.githubusercontent.com/73777251/104731775-360a7a80-5762-11eb-8a86-b50d98b58b01.png",
     "issueNumber": 247
   }
+
```

Comment:
I think we should remove this space 🤓

Reply:
Just checking in. Could you remove this space on line 476?

Reply:
I also agree with this. 👍